data/links.py

```diff
@@ -64,5 +64,6 @@
     'kiyo':'Use Adblock: http://www.youtube.com/user/gottagokiyo',
     'raseri':'NUTL Content: http://www.youtube.com/user/SmogonRaseri',
     'teddeh': "Some dude who said he's a girl: http://www.youtube.com/user/MrTeddehh",
-    'disjunction':'This guy is bad: '
+    'disjunction':'This guy is bad: ',
+    'yoshizilla315': "NU lives every week!: https://www.youtube.com/channel/UCFyInjfBzT38TSAKgHVQgkg"	
```

Comment:
no

Reply:
the other stuff I don't mind but not this

Reply:
worth a shot lol